frontend/public/components/modals/expand-pvc-modal.tsx

```diff
@@ -64,9 +64,10 @@ const ExpandPVCModal = withHandlePromise((props: ExpandPVCModalProps) => {
       <ModalBody>
         <p>
           <Trans t={t} ns="modal">
-            Increase the capacity of claim{' '}
+            Increase the capacity of PVC{' '}
```

Comment:
@andybraren can we use PVC here? Or should we use `PersistentVolumeClaim`?

Reply:
Hm, I don't think we have a writing convention around this yet. Since `PersistentVolumeClaim` is written out once in the modal title I'm cool with using either `PVC` or the original `claim` in the body to keep it shorter - whichever is more commonly used elsewhere in Storage/OCS.